src/Utils/MixedJSONHandler.php

```
@@ -39,12 +39,8 @@ public static function getSubscribingMethods(): array
     /** @phpstan-ignore-next-line */
     public function serializeMixedToJson(JsonSerializationVisitor $visitor, mixed $any, array $type, Context $context): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$visitor'. (undefined)

(UnusedFormalParameter)